easybuild/easyconfigs/m/mmtf-cpp/mmtf-cpp-1.0.0-GCC-10.3.0.eb

```diff
@@ -0,0 +1,38 @@
+# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2021/04
+easyblock = 'CMakeNinja'
+
+name = 'mmtf-cpp'
+version = '1.0.0'
+
+homepage = 'https://mmtf.rcsb.org/'
+description = """
+The Macromolecular Transmission Format (MMTF) is a new compact binary format to 
+transmit and store biomolecular structures for fast 3D visualization and       
+analysis.
+"""
+
+toolchain = {'name': 'GCC', 'version': '10.3.0'}
+
+source_urls = ['https://github.com/rcsb/mmtf-cpp/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['881f69c4bb56605fa63fd5ca50842facc4947f686cbf678ad04930674d714f40']
+
+builddependencies = [
+    ('CMake', '3.20.1'),
+    ('Ninja', '1.10.2'),
+    # ('Doxygen', '1.8.20'),
+]
+
+dependencies = [
+    ('msgpack-cxx', '4.0.2'),
+]
+
+# Ignore installing docs
+# postinstallcmds = ['cd ../%(name)s-%(version)s/docs && doxygen  && cd .. && mv docs %(installdir)s/doc']
```

Comment:
@akesandgren Any particular reason for this? Did you run into trouble here?

Reply:
Could have been a missing Doxygen at the time and I just didn't bother. Fixed now.